lib/builder/ProjectBuildContext.js

```diff
@@ -39,6 +39,10 @@ class ProjectBuildContext {
 		return this._project === this._buildContext.getRootProject();
 	}
 
+	getOption(key) {
```

Comment:
Should we rename this function to `getBuildContextOption` or is this fine?

Reply:
I think it's fine. It's anyway a private class.